editor/editor_properties.cpp

```diff
@@ -390,13 +390,36 @@ EditorPropertyCheck::EditorPropertyCheck() {
 
 void EditorPropertyEnum::_option_selected(int p_which) {
 
-	emit_signal("property_changed", get_edited_property(), p_which);
+	String text = options->get_item_text(p_which);
+	Vector<String> text_split = text.split(":");
+	if (text_split.size() == 1) {
+		emit_signal("property_changed", get_edited_property(), p_which);
```

Comment:
Did you miss a return after this statement?

Reply:
Yeah, thanks for catching that. I really shouldn't code at the end of a long day. I'll fix it.